contents/handbook/growth/marketing/index.md

```diff
@@ -37,18 +37,220 @@ To incentivise YC startups to choose PostHog for the product analytics, we'll of
+
+### Enterprise early adopters
+
+We'll forge deep relationships with our early enterprise customers who will be critical to develop our early enterprise product marketing and reaching more traditional enterprise engineering teams.
+
+We're interested to learn about their processes, their awareness and adoption of product analytics solutions, enterprise specific solutions, pricing and technical and compliance requirements.
+
+Initial user interviews will be informal, but as we start to identify key themes, will become more structured - making it easier for us to track improvements in our marketing. For example, early discussion may identify that non-technical teams choose product analytics solutions more often than technical teams.
+
+
+## Plan
+
+The plan for each quarter will be expanded closer to the time and issues will created and assigned to the team.
+
+### February
+
+- [x] Sean joined as mid-Feb
+- [x] Marketing strategy and campaigns, plan and functions
+- [ ] Product Marketing
+  - [ ] Conduct rapid market and competitor analysis
+  - [ ] Summarise existing user interviews
+  - [ ] Create feature page template
+  - [ ] Create product marketing for example feature for Design team
+- [ ] Hiring
+  - [x] Post Content Marketer role
+  - [ ] Post Product Marketer role
+
+### March
+
+- [ ] Campaign: Improved YC deal for S21
+  - [ ] Brainstorm ideas:
+    - YC S21 Application Support AMA, [example](https://dutchtechsf.com/events/ycombinator-s21)
+  - [ ] Plan campaign and define metrics
+  - [ ] Start executing, creating content and events
+  - Notable dates:
+    - [23 March 2021 YC W21 Demo Day](https://blog.ycombinator.com/yc-w21-virtual-demo-day/)
+- Product Marketing
+  - [ ] Continue creating product marketing for existing features
+- Content Marketing:
+  - [ ] Work with Design to improve release posts
+  - [ ] Syndicate releases to startup/hacker communities
+- Improve release newsletter:
+  - [ ] Spam detection
+  - [ ] Opens
+  - [ ] Clicks
+- Conduct a handful of conversion experiments:
+  - [ ] E.g changing CTA text, adding more CTAs, SEO etc
+- Metrics:
+  - [ ] Implement dashboards for conversion metrics, then make a start on remaining metrics
+  - [ ] Define growth goals for Q2
+- Hiring:
+  - [ ] Sign Content Marketer
+  - [ ] Sign Product Marketer
+
+### Q2
+
+- Product marketing
+  - Experiment changing existing messaging to improve conversion
+  - Expand market and competitor research
+  - Identify more personas
+  - Create product marketing for new features
+  - Start creating content for next personas
+- Content marketing
+  - Work with product marketing to create content themes that address buyer personas
+  - Classify existing content into themes and rank effectiveness
+  - Experiment within popular and new (e.g case studies) content themes
+  - Create a content schedule
+- Community
+  - Make it easier to contribute to in-the-monent and long-running discussions using open tools
```

Comment:
typo - "moment"

Reply:
Curious what you mean by this line - an example or two would be handy for me to understand

Reply:
Sure, in-the-moment would be real-time chat, where long running would be a discussion on a forum/issue which could span days, weeks or even months. Tools would be Slack/Matrix/Mattermost/Rocket.Chat and GitHub Issues/Discourse/PHPbb etc.